vuepress/docs/v7.3/tutorials/solution/install-standard-demo.md

```diff
@@ -47,41 +47,36 @@ Install the Standard Banking Demo by integrating the Entando Cloud Hub into your
 4. **From the Hub Catalog, `Deploy` and `Install` each of the four Standard Banking Demo bundles:**
 
    ::: warning
-   **The order of installation is important.** The `standard-demo-content-bundle` must be installed last, as it relies on MFEs from the other bundles to set up each page. To uninstall the Banking demo, the `standard-demo-content-bundle` must be uninstalled first to resolve dependencies before other bundles can be be removed.
+   **The order of installation is important.** The `sd-content-bundle` must be installed last, as it relies on MFEs from the other bundles to set up each page. To uninstall the Banking demo, the `sd-content-bundle` must be uninstalled first to resolve dependencies before other bundles can be be removed.
```

Comment:
**The order of installation is important.** `sd-content-bundle` must be installed last, as it relies on MFEs from the other bundles to set up each page. To uninstall the Banking demo, `sd-content-bundle` must be uninstalled first to resolve dependencies before other bundles can be be removed.

Never noticed, MFE is used here without first noting the the initialism above, at the first mention of micro frontends.

Can't comment on lines 10 & 11 above, but should add the initialisms there.